lizrd/grid/grid.py

```diff
@@ -99,11 +99,28 @@ def create_subprocess_args(
                 ], interactive_debug_session
 
             n_job_repetitions = 1
-            if "repeater_mode" in training_args and training_args["repeater_mode"]:
+            if (
+                "checkpoint_manager" in training_args
+                and training_args["checkpoint_manager"]
+            ):
                 total_exp_time = timestr_to_minutes(setup_args["time"]) * 60
                 if CLUSTER.max_exp_time < total_exp_time:
                     n_job_repetitions = ceil(total_exp_time / CLUSTER.max_exp_time)
-                    setup_args["time"] = seconds_to_timestr(CLUSTER.max_exp_time)
+                if (
+                    "scheduler_trapezoidal_slides" in training_args
+                    and training_args["scheduler_trapezoidal_slides"]
+                ):
+                    scheduler_trapezoidal_slides = literal_eval(
+                        training_args["scheduler_trapezoidal_slides"]
+                    )
+                    n_slide_jobs = 0
+                    for e in scheduler_trapezoidal_slides:
```

Comment:
Why `e`? Can we rename it to something more informative?

Reply:
Here and in other places as well

Reply:
done